framework/src/transfers/MultiAppShapeEvaluationTransfer.C

```diff
@@ -370,6 +370,7 @@ MultiAppShapeEvaluationTransfer::transferVariable(unsigned int i)
         if (node->n_dofs(sys_num, var_num) < 1)
           continue;
 
+        const auto node_id = node->id();
```

Comment:
What's going on here?  Is this an unused variable?

Reply:
haha yea that's why CIVET was screaming. It was a temporary debugging line since gdb appears to choke on condition breakpoints involving a call through a range-based for iterator (pointing to a `Node *` in this case)